Dockerfile

```diff
@@ -1,25 +1,31 @@
+FROM maven:3.6.1-jdk-8 AS tls-scanner-build
```

Comment:
Also, tls-scanner-build maven image is not deleted itself, we need to clean it up somehow:
```
grinder-framework                latest              6c150955dfe9        27 minutes ago      421MB
<none>                           <none>              25f8176dd2ea        48 minutes ago      767MB
```
I think that we don't need that 767MB image after building of Grinder is finished. Maybe we can clean it up, or am I wrong? We need to discuss it.